randovania/game_connection/connector/prime_remote_connector.py

```diff
@@ -7,6 +7,7 @@
 from typing import TYPE_CHECKING
 
 from open_prime_rando.dol_patching import all_prime_dol_patches
+from ppc_asm import assembler  # type: ignore[import-untyped]
```

Comment:
Why did this get moved out of the type checking block?

Reply:
line 39, `PatchInstructions = list[assembler.BaseInstruction]`

Reply:
ah, turns out I can put those definitions inside the type checking block! neat. hopefully nobody ever has to `cast()` to these types tho